crates/wasm-test/src/lib.rs

```diff
@@ -0,0 +1,60 @@
+use extism_pdk::*;
+use proto_plugin_sdk::*;
+
+#[plugin_fn]
+pub fn get_metadata(_: ()) -> FnResult<Json<PluginMetadata>> {
```

Comment:
So as long as the plugin functions are named like this, it will work?

Do we check if these functions are around when we load the wasm plugin?